Makefile

```diff
@@ -45,6 +45,8 @@ deploy-appbundle:
 		$(OUTPUT_DIR)/analytics-ga.js \
 		$(BUILD_DIR)/analytics-ga.min.js \
 		$(BUILD_DIR)/analytics-ga.min.map \
+		$(BUILD_DIR)/streamEffect.min.js \
```

Comment:
I think streamEffect is very generic. Isn't it? If we want to add another effect what will be its name?

You should change the name to something more specific. How about videoBlurEffect?

Reply:
Changed name to video-blur-effect